quantum/nodes/cnot-gate/cnot-gate.js

```diff
@@ -0,0 +1,47 @@
+module.exports = function (RED) {
+    "use strict";
+
+    function CNotGateNode(config) {
+      // Creating node with properties and context
+      RED.nodes.createNode(this, config);
+      this.name = config.name;
+      this.outputs = config.outputs;
+      const globalContext = this.context().global;
+      const node = this;
+      const output = new Array(node.outputs);
+
+      //Connect gate to 2 quantum registers as input, indicating which input are the controls and which is the target
+      //Add the inputs to the qiskit code 'qc = cx(qr1, qr2)' and append to the script
+      //Output qr1 and qr2 from gate.
+
+      this.on("input", function (msg, send, done) {
+        // Appending Qiskit script to the 'script' global variable
+        let qiskitScript =
+          "\nqc = cx(" +
+          document.getElementById("control-qubit-1").value +
```

Comment:
I don't think you can access DOM objects from backend. You can get the same control qubit value from config variable passed to CNotGetNode function, and save it locally like this this.control-qubit-1 = config.control-qubit-1. And I think you also wanna make sure you name you inputs like "node-input-<property_name> inside html file.